components/consentRequestForm/dateInput/index.jsx

```diff
@@ -41,38 +40,29 @@ export const TESTCAFE_ID_DATE_PICKER_CALENDAR_BUTTON =
 const useStyles = makeStyles((theme) => createStyles(styles(theme)));
 
 export default function DateInput(props) {
-  const {
-    selectedDate,
-    setSelectedDate,
-    datepickerOpen,
-    setDatepickerOpen,
-    handleDateChange,
-    setDateForever,
-  } = props;
+  const { selectedDate, handleDateChange, setDateForever } = props;
```

Comment:
doesn't the linter tell you to do this in the function declaration?

```
export default function DateInput({ selectedDate, handleDateChange }) {
```

Reply:
Note, I removed `setDateForever` as `handleDateChange` should just return a string

Reply:
Huh I thought I changed this... I will change now.